cfme/v2v/conversion_hosts.py

```diff
@@ -0,0 +1,239 @@
+import attr
+from navmazing import NavigateToAttribute
+from navmazing import NavigateToSibling
+from widgetastic.utils import WaitFillViewStrategy
+from widgetastic.widget import TextInput
+from widgetastic.widget import View
+from widgetastic_patternfly import BootstrapSelect
+from widgetastic_patternfly import Button
+from widgetastic_patternfly import Input
+from widgetastic_patternfly import Text
+
+from cfme.base.login import BaseLoggedInPage
+from cfme.infrastructure.provider.rhevm import RHEVMProvider
+from cfme.modeling.base import BaseCollection
+from cfme.modeling.base import BaseEntity
+from cfme.utils.appliance.implementations.ui import CFMENavigateStep
+from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.utils.appliance.implementations.ui import navigator
+from cfme.utils.wait import TimedOutError
+from cfme.utils.wait import wait_for
+from widgetastic_manageiq import ConversionHostProgress
+from widgetastic_manageiq import HiddenFileInput
+from widgetastic_manageiq import SearchBox
+from widgetastic_manageiq import V2VBootstrapSwitch
+from widgetastic_manageiq import WaitTab
+
+
+class MigrationSettingsView(BaseLoggedInPage):
+    configure_conversion_host = Text(locator='(//a|//button)[text()="Configure Conversion Host"]')
+    conversion_host_progress = ConversionHostProgress()
+
+    @property
+    def in_migration_settings(self):
```

Comment:
Is it intended that this View class will be used as a base class for other migration views? I see currently its being used directly for a navigation destination.  If its not intended to be a base class for inheritance, then this can be implemented directly in `is_displayed`.